### PR TITLE
Design 335: add nuxeo data table to storybook

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

**Comment by guirenard** [comment]
Probably to be rediscussed but, we normally commit with an ELEMENTS ticket on this repo.

File: storybook/package.json [hunk 1]
```diff
@@ -8,10 +8,15 @@
   "private": true,
   "dependencies": {
     "@babel/core": "^7.3.4",
+    "@nuxeo/nuxeo-elements": "^3.0.0-SNAPSHOT.20190603.162100-9fc5041",
```
Comment (guirenard) [inline storybook/package.json]: Why not pulling latest?

Comment (Gabez0r) [reply storybook/package.json]: Dependencies on elements should be set to "3.0.0-SNAPSHOT". Please check nuxeo/nuxeo-elements#129.

**Comment by semisse** [comment]
Moving PR's to ELEMENTS

**Comment by semisse** [comment]
Moved into https://github.com/nuxeo/nuxeo-elements/pull/131

**Pull request closed without merging**